spec/openapi.yaml

```diff
@@ -1138,6 +1160,66 @@ components:
                 description: Path to package
                 example: "jo/packages"
 
+    Job:
+      type: object
+      properties:
+        id:
+          type: integer
+          example: 420
+        name:
+          type: string
+          description: Short Filename
+          example: "SPICE~1.gco"
+        display_name:
+          type: string
+          description: Long Filename
+          example: "Spice_Harvester_0.3mm_PLA_MK3S_12m.gcode"
+        path:
+          type: string
+          example: "/local"
+        display_path:
+          type: string
+          example: "/PrusaLink gcodes"
+        size:
+          type: integer
+          description: Bytes
+          example: 2393142
+        m_timestamp:
+          type: integer
+          description: Timestamp in seconds
+          example: 1648042843
+        state:
+          type: string
+          enum: [ IDLE, BUSY, PRINTING, PAUSED, FINISHED, STOPPED, ERROR, ATTTENTION, READY ]
```

Comment:
JOB can't be IDLE, BUSY or READY! It can be:

- PRINTING
- PAUSED (user's pause, or ATTENTION state)
- FINISHED
- STOPPED
- ERROR

Please see Connect job states, which are FIN_OK, FIN_ERR, etc..